Content.Server/Administration/Commands/ClearDisplayNameCommand.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Linq;
+using Content.Shared.Administration;
+using Robust.Server.Player;
+using Robust.Shared.Console;
+
+namespace Content.Server.Administration.Commands;
+
+[AdminCommand(AdminFlags.Admin)]
+public sealed class ClearDisplayNameCommand : LocalizedCommands
+{
+    [Dependency] private readonly IPlayerManager _playerManager = default!;
+
+    public override string Command => "cleardisplayname";
+    public override string Description => Loc.GetString("cmd-displayname-clear-description");
+    public override string Help => Loc.GetString("cmd-displayname-clear-help");
+
+    public override CompletionResult GetCompletion(IConsoleShell shell, string[] args)
+    {
+        if (args.Length == 1)
+        {
+            var names = _playerManager.Sessions.OrderBy(c => c.Name).Select(c => c.Name).ToArray();
+            return CompletionResult.FromHintOptions(names, LocalizationManager.GetString("shell-argument-username-optional-hint"));
+        }
```

Comment:
Can't you just pass an IEnumerable in? I don't think it even needs the full array alloc.